sh/generate_cluster_config.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+TEMPLATE_FILE=cluster-template.yaml
```

Comment:
I was thinking of using Ansible for its templating engine, and ditch this sh. Using ansible has a few advantages, as it gives us control loops that we can use, for example for iterating over a list of additional tags.

Reply:
Ansible is good when it comes to provisioning infrastructure. And my personal opinion is that is should not be meant to be used for things that it is not meant to do 🙈 
But I am open to discussion about this. And I did try to use ansible. But couldn't figure out a way to edit files. Would be interested to learn how to do it 😄